src/arfsdao.ts

```diff
@@ -691,6 +692,7 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 	async getAllFoldersOfPrivateDrive(
```

Comment:
Though my type safety branches will prevent issues here, I'd make this a deconstructed object parameter list.

Reply:
There it is: 8198d1c